addons/jest/src/index.ts

```diff
@@ -3,13 +3,17 @@ import deprecate from 'util-deprecate';
 import { normalize } from 'upath';
 import { ADD_TESTS } from './shared';
 
+interface AddonParameters {
+  jest?: string | string[] | { disable: true };
+}
+
 const findTestResults = (
   testFiles: string[],
   jestTestResults: { testResults: Array<{ name: string }> },
   jestTestFilesExt: string
 ) =>
   Object.values(testFiles).map(name => {
-    const fileName = `${name}${jestTestFilesExt}`;
+    const fileName = `[\\/\\\\]${name}${jestTestFilesExt}`;
```

Comment:
Can you give some context around this funny business?

Reply:
I have replaced regex with upath.sep.
I think this is more obvious and don't require code-comment.